scripts/style/shellcheck_skip.txt

```diff
@@ -9,6 +9,7 @@ image/templates/common/ca-setup.sh
 image/templates/common/delete-ca.sh
 image/templates/common/port-forward.sh
 image/templates/common/setup-central.sh
+image/templates/common/setup-scanner-v4.sh
```

Comment:
I think we shouldn't add new scripts here, instead we should address the shellcheck complaints. In the particular case it is about the go templating syntax in it. I guess we should simply add a `# shellcheck disable` pragma to `setup-scanner-v4.sh` and not add anything here.